src/Parser/DestructionWarlock/Modules/Features/CastEfficiency.js

```diff
@@ -0,0 +1,350 @@
+import SPELLS from 'common/SPELLS';
+
+import CoreCastEfficiency from 'Parser/Core/Modules/CastEfficiency';
+
+class CastEfficiency extends CoreCastEfficiency {
+  static CPM_ABILITIES = [
+    ...CoreCastEfficiency.CPM_ABILITIES,
+    //Rotational spells
+    {
+      spell: SPELLS.CHAOS_BOLT,
+      category: CastEfficiency.SPELL_CATEGORIES.ROTATIONAL,
+      getCooldown: haste => null,
+      recommendedCastEfficiency: 0.95,
```

Comment:
Cast efficiency can't really be set for abilities with no cooldown.

Reply:
Ah, fair enough, that makes sense.